json-guides/retry-timeout.json

```diff
@@ -91,6 +91,64 @@
                 }
             ]
         },
+        {
+            "name": "TimeoutAnnotation",
+            "title": "Adding the @Timeout annotation",
+            "description": [
+                "Now that you've seen the application easily fail when it loads for an indefinite amount of time, let's apply a Timeout policy to to limit the time the request is made to a transaction history and to prevent the page from loading forever.",
+                "With a Timeout policy, the microservice fail with <code>TimeoutException</code> after the set amount of time has passed when the request to a transaction history is made.",
+                "The <code>@Timeout</code> annotation has one parameter that specifies the amount of time it took to make a transaction history request to the service. If the parameter is not specified, the default is <code>1000</code> milliseconds.",
```

Comment:
Probably introduce this first before the `TimeoutException` in the paragraph above.

Reply:
The Timeout annotation takes a (long?) integer value that specifies the time allowed for the request to finish.